starcheck/src/lib/Ska/Starcheck/Dark_Cal_Checker.pm

```diff
@@ -1895,8 +1896,15 @@ package TLREntry;
 
 use strict;
 use Carp;
-use Ska::Convert qw(date2time);
```

Comment:
I think that Ska::Convert date2time and time2date both have issues and I'd like to replace them globally with the Chandra.Time methods, but it looks like subbing in just at this location fixes some issues with unexpected delta times when checking that the timing of the commands matches the template in the dark call checker.